packages/typedefs/src/interaction.ts

```diff
@@ -0,0 +1,85 @@
+
+export const UNKNOWN /*                       */ = "UNKNOWN"
+export const SCRIPT /*                        */ = "SCRIPT"
+export const TRANSACTION /*                   */ = "TRANSACTION"
+export const GET_TRANSACTION_STATUS /*        */ = "GET_TRANSACTION_STATUS"
+export const GET_ACCOUNT /*                   */ = "GET_ACCOUNT"
+export const GET_EVENTS /*                    */ = "GET_EVENTS"
+export const PING /*                          */ = "PING"
+export const GET_TRANSACTION /*               */ = "GET_TRANSACTION"
+export const GET_BLOCK /*                     */ = "GET_BLOCK"
+export const GET_BLOCK_HEADER /*              */ = "GET_BLOCK_HEADER"
+export const GET_COLLECTION /*                */ = "GET_COLLECTION"
+export const GET_NETWORK_PARAMETERS /*        */ = "GET_NETWORK_PARAMETERS"
+
+export const BAD /* */ = "BAD"
+export const OK /*  */ = "OK"
+
+export const ACCOUNT /*  */ = "ACCOUNT"
+export const PARAM /*    */ = "PARAM"
+export const ARGUMENT /* */ = "ARGUMENT"
+
+export const AUTHORIZER /* */ = "authorizer"
+export const PAYER /*      */ = "payer"
+export const PROPOSER /*   */ = "proposer"
+
+export interface IAcct {
```

Comment:
nit: I don't think prefixing interfaces with `I` is standard in TS.  And if we do prefix interfaces, it should be something that is consistent (currently our interfaces in @onflow/typedefs & FCL are not prefixed).

https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines (guidelines typescript team uses internally).  There used to be a statement about it on the official TypeScript Handbook too, I just can't seem to find it.